demo/src/components/CombinedFilters.js

```diff
@@ -15,6 +17,8 @@ const CombinedFilters = () => {
             transform={transform}
             rangeFacetItemComponent={<RangeFacetItemComponent />}
             textFacetItemComponent={<TextFacetItemComponent />}
+            multiLevelFacetItemComponent={<MultiLevelItemComponent />}
+            multiLevelfacetClick={multiLevelfacetClick}
```

Comment:
what about onClick for Text and Range facets?
Can we have a onFacetClick and handle clicks of 3 types of facets in that?

Reply:
Add this to prop types.

Reply:
How is `multiLevelfacetClick` being used?

Reply:
was not using the method, so removed it